srcpkgs/ImageMagick/template

```diff
@@ -1,28 +1,26 @@
 # Template file for 'ImageMagick'
 pkgname=ImageMagick
-version=7.0.11.12
-revision=3
+version=7.1.0.8
+revision=1
 _majorver=${version%.*}
 _patchver=${version##*.}
 wrksrc="${pkgname}-${_majorver}-${_patchver}"
 build_style=gnu-configure
-configure_args="--without-autotrace --with-wmf=yes
- --without-dps --without-fpx --without-gvc --without-jbig --with-gslib=yes
- --without-lqr --without-openexr --with-gs-font-dir=/usr/share/fonts/Type1
- --with-magick-plus-plus --with-modules --enable-shared --with-rsvg
- --with-dejavu-font-dir=/usr/share/fonts/TTF --enable-opencl --disable-static
- --with-openjp2"
+configure_args="--disable-static --enable-opencl --with-modules --with-gslib
+ --with-rsvg --with-wmf --with-dejavu-font-dir=/usr/share/fonts/TTF
+ --with-gs-font-dir=/usr/share/fonts/Type1"
```

Comment:
I unsdertand removing cruft, but some of the options can be useful. If they auto detect things we don't want, for example... Did you test the package was the same?